packages/monorepo-builder/packages/init/src/Composer/PackageNameVersionProvider.php

```diff
@@ -47,6 +47,20 @@ public function provide(string $packageName): string
         return sprintf('^%d.%d', $version->getMajor()->getValue(), $version->getMinor()->getValue());
     }
 
+    /**
+     * Workaround for when this is executed in the monorepo
+     */
+    private function getPrettyVersion(string $packageName): string
+    {
+        try {
+            return PrettyVersions::getVersion($packageName)
+                ->getPrettyVersion();
+        } catch (OutOfBoundsException | ReplacedPackageException $exception) {
+            return PrettyVersions::getVersion('symplify/symplify')
+                ->getPrettyVersion();
+        }
+    }
+
```

Comment:
This would be very hard to undrestand without any commet and link to the root cause issue (the one you linked could be good enough).

Also it hides dependency on `symplify/sypmlify` package. That should be at least optional parameter above.